### DISCUSSION TITLE
User registration & authentication.

### DISCUSSION THREAD

**Discussion by luowenhao-github** [elsa-workflows/elsa-core#4418] [Q&A]
Does the elsa dashboard include user registration & authentication now?
If no, when it will include.

**Comment by sfmskywalker** [comment]
Hello @luowenhao-github , The dashboard doesn't support this. Elsa Studio, the Elsa 3 dashboard app, will have a module to manage users, roles and permissions. As to when, that depends on many factors and is hard to predict.
